zulip/integrations/rocket.chat/README.md

```diff
@@ -0,0 +1,20 @@
+# Zulip <--> Rocket.Chat mirror
+
+## Usage
+
+0. `pip install zulip rocketchat_API`
+
+### 1. Zulip endpoint
+1. Create a generic Zulip bot
+2. (don't forget this step!) Make sure the bot is subscribed to the relevant stream
+3. Enter the bot's email and api_key into rocket_mirror_config.py
+4. Enter the destination subject and realm into the config file
+
+### 2. Rocket.Chat endpoint
+1. Create a user
+2. Enter the user's username and password into rocket_bridge_config.py
```

Comment:
Is it `rocket_bridge_config` or `rocket_mirror_config`?

Reply:
Whoops, my mistake it should be `rocket_mirror_config` (using Zulip's nomenclature).